packages/ui/core-components/src/lib/atoms/inputs/date-range/DateRange.svelte

```diff
@@ -39,6 +39,9 @@
 	/** @type {string | undefined} */
 	export let defaultValue;
 
+	/** @type {boolean} */
+	let range = true;
```

Comment:
Make sure we can accept the string 

```
<DateInput
  range=true
/>
```

not just 

range={true}

Reply:
See other components like 

BarChart labels=true for examples

Reply:
note the preferred syntax should just be `<DateInput range />` (which svelte turns into `<DateInput range={true} />`

Reply:
@csjh Thanks, Sounds good. Docs are set as <DateInput range /> for the examples.
@archiewood added the ToBoolean check for the range prop to accept strings and booleans